tools/gen_node_cert.sh

```diff
@@ -318,13 +319,37 @@ main()
         cat ${gmkey_path}/gmagency.crt >> ${output_dir}/${gm_conf_path}/gmnode.crt
 
         #move origin conf to gm conf
-        rm ${output_dir}/${conf_path}/node.nodeid
+        # rm ${output_dir}/${conf_path}/node.nodeid
         cp -r ${output_dir}/${conf_path} ${output_dir}/${gm_conf_path}/origin_cert
         #remove original cert files
         rm -rf ${output_dir:?}/${conf_path}
         mv ${output_dir}/${gm_conf_path} ${output_dir}/${conf_path}
     fi
+    if [[ -n "${sdk_cert}" ]]; then
+        if [ -n "$guomi_mode" ]; then
+            mv ${output_dir}/${conf_path}/gmnode.key ${output_dir}/${conf_path}/gmsdk.key
+            mv ${output_dir}/${conf_path}/gmnode.crt ${output_dir}/${conf_path}/gmsdk.crt
+            mv ${output_dir}/${conf_path}/gmennode.key ${output_dir}/${conf_path}/gmensdk.key
+            mv ${output_dir}/${conf_path}/gmennode.crt ${output_dir}/${conf_path}/gmensdk.crt
+            mv ${output_dir}/${conf_path}/gmca.crt ${output_dir}/${conf_path}/gmca.crt
+            mv ${output_dir}/${conf_path}/gmnode.nodeid ${output_dir}/${conf_path}/gmsdk.publickey
+            mv ${output_dir}/${conf_path}/origin_cert/node.key ${output_dir}/sdk.key
+            mv ${output_dir}/${conf_path}/origin_cert/node.crt ${output_dir}/sdk.crt
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5833446)